how_to_contribute.md

```diff
@@ -0,0 +1,51 @@
+# Quay-logs - Contributing/Docs
```

Comment:
This is a kind of docs (documentation of code)!!

Reply:
Can you please add `CONTRIBUTING.md or HOW_TO_CONTRIBUTE.md` mentioning the steps like `https://github.com/openebs/maya/blob/master/CONTRIBUTING.md#steps-to-contribute`  in another PR?

You should rename the current file name to `docs.md`.

Reply:
The confusion was. In the last meeting amit sir said to write down my understanding in how_to_contribute. Since then while writing it I too thought it more of as a documentation rather than how to contribute. That's why made its heading as contributing/docs.